concepts/option/about.md

```diff
@@ -1 +1,21 @@
-TODO: add information on option concept
+The `Option` type is an enum provided by the standard library. It is useful for expressing whether a value exists or does not exist.
+While this may sound like `null` or `nil`, it is quite different and special.
+`Option` is distinctive to Rust. It is also quite fun to work with!
```

Comment:
> `Option` is distinctive to Rust.

Not true. For example, [`Maybe` in Haskell](https://wiki.haskell.org/Maybe). I'm pretty sure it exists in the ML family of languages also.

Might be better to phrase as something like "`Option` builds on language design from functional languages." Or maybe just omit that sentence.

Reply:
Good to know; Thanks! I went with omitting for now.